pages/instructorAssessmentInstances/instructorAssessmentInstances.js

```diff
@@ -105,6 +105,14 @@ router.post('/', function(req, res, next) {
                 res.redirect(req.originalUrl);
             });
         });
+    } else if (req.body.__action == 'grade_all') {
+        const assessment_id = res.locals.assessment.id;
+        const close = false;
```

Comment:
Why is this not also closing the assessment instances? For per-instance actions we only have the "Grade & Close" action, so I would have thought that the "all" button would mimic this. Or are you thinking ahead to #3622?

Reply:
I was thinking that the "Close All" could be achieved by expiring the time limit for all assessments, which would work for now and for after #3622 is implemented, but I don't have a problem with changing this to also close all assessments. The idea was to give instructors the ability to grade them without closing, and then closing them separately. I can also have both options ("Grade All" and "Close All"), and add the Grade only button in per-instance actions.

Reply:
The problem with expiring the time limit for all assessments is that it doesn't work for Homework-type assessments, since they don't check the time limit.

Reply:
I added both options ("Grade All" and "Close All").